core/jobs/batch_jobs/exp_version_history_computation_job.py

```diff
@@ -101,13 +125,13 @@ def generate_exploration_from_snapshot(
                 return None
 
     def convert_to_formatted_model_group(
-        self, model_group: UnformattedModelGroupDict
-    ) -> Optional[FormattedModelGroupDict]:
+        self, model_group: UnformattedModelGroupForVerificationJobDict
+    ) -> Optional[FormattedModelGroupForVerificationJobDict]:
         """Returns True if the given model group is valid.
 
         Args:
-            model_group: UnformattedModelGroupDict. The model group to be
-                checked.
+            model_group: UnformattedModelGroupForVerificationJobDict.
+                The model group to be checked.
 
         Returns:
             bool. Whether the given model group is valid or not.
```

Comment:
In the function signature, you are mentioning `Optional[FormattedModelGroupForVerificationJobDict]` whereas in return statement you are saying bool.

Reply:
Done.